WordPressKit/WordPressKit/ReaderServiceProtocol.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+
+
+/// Base Subscriptable protocol
+public protocol ReaderServiceSubscriptable {
```

Comment:
The idea is to use these 3 protocol with a `ReaderTopicService` extension

Reply:
Unless you actually plan to actually swap instances in any spot (which would justify having a protocol), i'd suggest just nuking the protocol definition, since, otherwise, it wouldn't add value.

Plus: none of the other services is defined that way.

Please, don't get me wrong: OCMock can instantiate mockups based on protocols, which is neat for unit testing. *But* we're not using it to test services... and it's tied up to ObjC.